python.html.markdown

```diff
@@ -635,6 +635,21 @@ def create_adder(x):
 add_10 = create_adder(10)
 add_10(3)   # => 13
 
+# Closures in nested functions, the nonlocal keyword to work with variables in nested scope which shouldn't be in the inner functions.
```

Comment:
Prefer a statement, like:
```python
# Closures in nested functions:
# We can use the nonlocal keyword to work with variables in nested scope which shouldn't be declared in the inner functions.
```

Reply:
Addressed in https://github.com/adambard/learnxinyminutes-docs/pull/4033/commits/28b4f73f1bc3055df8b0ca00dc49e93d0ced2f20